transcom/mymove
# MAIN B-19807 fe sc ability to edit departure aoa

## [B-19807](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19807)

INT PR 1: https://github.com/transcom/mymove/pull/12837
INT PR 2: https://github.com/transcom/mymove/pull/12962

## Summary

Added an edit icon button to `Actual Move Date` and `Advance Received` on Review Documents page when reviewing as a closeout SC. The edit button opens up a Modal where you can edit values of these.

### How to test

1. Use `MoveWithPPMShipmentReadyForFinalCloseout` testharness to create a move.
2. As a customer -> go to the move and complete uploading PPM documents.
3. As a closeout SC -> find the move and click "Review Documents".
4. Click "Show details" arrow button.
5. You should see a pencil icon next to "Actual Move Date" -> Click on it.
6. Change the value in the modal and save.
7. Should see the change reflected.
8. Go to the last page of the review before confirm.
9. Click "Show details" for "Incentives/Costs" and do the same test for "Advance Received" button.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/1b6e0d7c-5379-4fe7-8184-ea69a979234c)
